config/initializers/02_urls.rb

```diff
@@ -11,7 +13,8 @@
 FEATURE_UPVOTE_URL = ENV.fetch("FEATURE_UPVOTE_URL", "https://demarches-simplifiees.featureupvote.com")
 
 # Internal URLs
-FOG_BASE_URL = "https://static.demarches-simplifiees.fr"
+FOG_OPENSTACK_URL = ENV.fetch("FOG_OPENSTACK_URL", "https://static.demarches-simplifiees.fr")
+DS_PROXY_URL = ENV.fetch("DS_PROXY_URL")
```

Comment:
Pour raccrocher les wagons avec la PR https://github.com/betagouv/demarches-simplifiees.fr/pull/6932 :

```suggestion
FOG_OPENSTACK_URL = ENV.fetch("FOG_OPENSTACK_URL", "https://auth.cloud.ovh.net")
DS_PROXY_URL = ENV.fetch("DS_PROXY_URL", "https://static.demarches-simplifiees.fr")
```

Reply:
J'ai modifié `FOG_OPENSTACK_URL` pour avoir une valeur par défaut correcte.

(En revanche DS_PROXY_URL, pour l'instant, je m'en sers pour savoir si l'appli utilise un proxy tout court, donc je garde une valeur par défaut vide)